.github/workflows/test-all-packages.yml

```diff
@@ -468,6 +468,11 @@ jobs:
     strategy:
       matrix:
         engine: ['18.x', '20.x', 'xs']
+        testglob:
```

Comment:
Ava has a feature built-in for this. Please use that instead https://github.com/avajs/ava/blob/main/docs/recipes/splitting-tests-ci.md#splitting-tests-on-github-actions

Reply:
Oh, very cool!  Thanks, I will try that.